scripts/update-w3c-groups-list.js

```diff
@@ -43,6 +45,16 @@ async function update() {
     data[type][shortname] = { id, name, URI: url };
   }
 
+  // Exceptional groups that don't follow norms like other group types.
+  const miscGroupFilters = [group => group.shortname === "tag"];
+  for (const filter of miscGroupFilters) {
+    const group = json._embedded.groups.find(filter);
+    if (!group) continue;
+    const { shortname, id, name, _links: links } = group;
+    const url = links.homepage?.href;
+    data["misc"][shortname] = { id, name, URI: url };
+  }
+
```

Comment:
I hope this is not too much of an abstraction :smile:

Reply:
Maybe should just add exception for TAG..

Reply:
Hmm... the TAG is a weird group. They don't have any patent protection because the are not allowed to publish specs. They even have their own special kind of publication types called "findings". 

I don't mind keeping this though.

Reply:
The ethical-web-principles repo had https://www.w3.org/2001/tag/disclosures as `wgPatentURI` though.

Reply:
That points to an obsolete page. As the TAG doesn't publish specs, I don't think wgPatentURI applies still.